specs/RuntimeSpec/Fees/Fees.md

```diff
@@ -73,7 +75,26 @@ Here is the list of actions and their corresponding fees:
     - the base fee [`delete_key_cost`](/GenesisConfig/RuntimeFeeConfig/ActionCreationConfig.md#delete_key_cost)
 - [DeleteAccount](/RuntimeSpec/Actions.md#deleteaccountaction) uses
     - the base fee [`delete_account_cost`](/GenesisConfig/RuntimeFeeConfig/ActionCreationConfig.md#delete_account_cost)
-
+    - action receipt creation fee for creating Transfer to send remaining funds to `beneficiary_id`
+    - full transfer fee described in the corresponding item
+    
+## Charging fees
+
+In `Runtime`, fees are tracked in the `ActionResult` struct containing data about burnt and used gas. 
```

Comment:
This section only talks about receipts, but we start charging fees when transactions are converted to receipts. Please document that as well (if it is not already done).